containers/python/Dockerfile

```diff
@@ -5,6 +5,7 @@ COPY requirements_full.txt requirements.txt
 
 # Install the project dependencies
 RUN pip install -r requirements.txt
+RUN apt-get update && apt-get install -y procps && rm -rf /var/lib/apt/lists/*
```

Comment:
I think that this install would deserve a comment indicating why it is needed as this is not obvious:
Ex:  `Required by nextflow`

Reply:
I recommend adding a version to the procps package for reproducibility, stability, and clarity. While the risk of behavior changes for a basic package is low, it is still present. Specifying a version will also set a good example for others to do the same when adding extra packages.